pandas/core/indexes/base.py

```diff
@@ -3396,8 +3396,11 @@ def map(self, mapper, na_action=None):
 
     def isin(self, values, level=None):
         """
+        Return a boolean array where the index values are in `values`.
```

Comment:
Do you know where these changes are coming from? Ideally this PR will just do the deprecation.

Reply:
@TomAugspurger - Appears they snuck in with that large pd file under commit 4a43815d97811e1d4da78f495df3180e09eae7aa (GH20249)